.vscode/launch.json

```diff
@@ -616,6 +616,21 @@
       ],
       "outputCapture": "std"
     },
+    {
+      "name": "export-gltf",
+      "presentation": {
+        "group": "6_Misc"
+      },
+      "cwd": "${workspaceFolder}/test-apps/export-gltf",
+      "type": "node",
+      "request": "launch",
+      "program": "${workspaceFolder}/test-apps/export-gltf/lib/ExportGltf.js",
+      "args": [   // customize your input/output below
+        "--input=d:\\triangleWithUVParams.bim",
```

Comment:
is this iModel and gltf somewhere in the repo?

Reply:
I'd suggest removing that from the args committed. Otherwise PR looks good.

Reply:
Thanks. Neither are in the repo, but I think it needs a placeholder to show how to customize. What about "d:\\foo.bim" and "d:\\foo.gltf"?

Reply:
using "foo" or "replaceme" as the name is good with me.